marklogic-data-hub-central/ui/src/pages/Run.tsx

```diff
@@ -51,6 +51,9 @@ const Run = (props) => {
         retryLimit: 10  // Timeout after retries
     }
 
+    useEffect(() => {
+        return () =>  Modal.destroyAll()
+    }, []);
```

Comment:
any reason this is applied only to run.tsx. does curate/model/load tile modals get destroyed on log off?

Reply:
I tested the other tile modals, but I only saw the issue on the run page.